.gitignore

```
@@ -12,5 +13,7 @@ CHANGELOG.md
 *.bisign
 *.tar.gz
 *.zip
+hemtt*
+!hemtt.toml
```

```suggestion
./.hemtt
```
This should gitignore the template folder but not the `toml` file.